.github/workflows/sdk.yml

```diff
@@ -22,36 +22,66 @@ concurrency:
 
 jobs:
   go:
+    uses: ./.github/workflows/_sdk_check.yml
+    with:
+      sdk: go
+  go-dev:
```

Comment:
When running on main, do we still want to run both `<sdk>-dev` and `<sdk>` jobs?